src/main/java/it/gov/pagopa/pu/sil/util/Utilities.java

```diff
@@ -59,4 +59,10 @@ public static LocalDate getSpontaneousSilExpirationDate() {
       .isEqual(now.toLocalDate());
     return now.toLocalDate().plusDays(isSameDay ? 0 : 1);
   }
+
+  public static boolean isNaturalPerson(String fiscalCode) {
```

Comment:
let's add a javadoc to explain we are checking if it's a {@link it.gov.pagopa.pu.debtpositions.dto.generated.PersonEntityType#F}